### PR TITLE
[17.0][MIG] crm_partner_company_group: Migration to 17.0

### PR DESCRIPTION
**Pull request by MohamedOsman7**

### CODE REVIEW

**Review by CRogos — APPROVED** [review]
Code + Functional review LGTM

File: crm_partner_company_group/i18n/partner_company_group.pot
Comment (CRogos) [inline crm_partner_company_group/i18n/partner_company_group.pot:1]: I think this file can be deleted.

**Comment by CRogos** [comment]
@pedroguirao, @Christian-RB could you make a review on these *_company_group modules?

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**